src/commands/testing/spec/testing.spec.server.ts

```diff
@@ -185,7 +197,8 @@ testteardown,tests/testteardown.sas,sasjs_test_id,not provided,
             {
               test_loc: 'tests/testsetup.sas',
               sasjs_test_id: '',
-              result: 'not provided'
+              result: 'not provided',
+              test_url: `https://sas.analytium.co.uk/SASJobExecution/?_program=/Public/app/cli-tests/${target.name}/tests/testsetup&_debug=2477`
```

Comment:
perhaps the server should be a constant somewhere?  Else it will be challenging for other contributors to run the tests in their own environment